easybuild/easyblocks/generic/systemcompiler.py

```diff
@@ -115,9 +115,33 @@ def extra_options():
         return extra_vars
 
     def __init__(self, *args, **kwargs):
-        """Extra initialization: determine system compiler version and prefix."""
+        """Extra initialization: keep track of values that may change due to modifications to the version."""
         super(SystemCompiler, self).__init__(*args, **kwargs)
 
+        # fix installdir and module names (may differ because of changes to version)
```

Comment:
@ocaisa This doesn't make sense (anymore?), since we didn't touch `self.cfg['version']` at all? Neither did we before you reshuffled things (we only do in `make_module_step` and `make_module_extend_modpath`, but neither of those affect the module name or the install directory?

Reply:
I think I was just preserving what was originally there, I need to dig a bit to understand the issue. Do you mean that all of this can be moved to `make_module_step`?

Reply:
Ok, I think you are right, I can't see any any reason for the mns related stuff, we only need `self.orig_version` and `self.orig_installdir`

Reply:
Done